src/AppBundle/Controller/ArticleInfoController.php

```diff
@@ -121,13 +121,22 @@ public function gadgetAction(Request $request)
 
     /**
      * Display the results.
-     * @Route("/articleinfo/{project}/{article}", name="ArticleInfoResult", requirements={"article"=".+"})
+     * @Route(
+     *     "/articleinfo/{project}/{start}/{end}/{article}", name="ArticleInfoResult",
```

Comment:
This routing change will be a problem :( ArticleInfo is very popular and there are a lot of links to it. As optional parameters it'd be better if the start/end dates are at the end, but I suppose there's the caveat that "My article/2017-01-23/2018-01-23" is a valid page title.

Maybe we should use query string parameters instead?

Reply:
Ideally we could use some trickery to handle the "My article/2017-01-23/2018-01-23" edge case, or perhaps we don't care... Interested to hear your thoughts.